reflect/src/test/java/io/spine/reflect/InvokablesTest.java

```diff
@@ -117,38 +114,38 @@ void throwOnInvocationError() throws NoSuchMethodException {
         @DisplayName("convert a visible method to a handle")
         void convertToHandle() throws Throwable {
             var method = ClassWithPrivateMethod.class.getMethod("publicMethod");
-            var handle = asHandle(method);
-            assertThat(handle).isNotNull();
+            var handle = Invokables.asHandle(method);
+            Truth.assertThat(handle).isNotNull();
```

Comment:
Looks like imports were badly optimized. Static imports are in place, but seem to be not used.